pkg/trace/agent/agent.go

```diff
@@ -294,7 +294,7 @@ func (a *Agent) ProcessStats(in pb.ClientStatsPayload, lang string) {
 				Distributions:    make(map[string]stats.Distribution),
 				ErrDistributions: make(map[string]stats.Distribution),
 			}
-			aggr := stats.NewAggregation(out.Env, b.Resource, b.Service, "", statusCode, in.Version)
+			aggr := stats.NewAggregation(out.Env, b.Resource, b.Service, "", statusCode, in.Version, false)
```

Comment:
Now that I think of it, the synthetics aggregation should also be handled by the new stats endpoint like any other aggregation. But that requires to add it to the payload format. What do you think @furmmon?

Reply:
Can you share more details about what you're thinking? I want to get this in before the next code freeze - if possible.

Reply:
As discussed offline, let's do it in a follow up PR.